README.md

```diff
@@ -91,11 +91,11 @@ The integration tests don't run by default. To run all the tests, including inte
 
 The tests needs a MySQL server with replication enabled.
 
-Ecco includes a Docker Compose definition that can be used for this.
+Ecco includes a docker-compose solution that can be used for this. The file `docker-mysql-base.yml` is used together with one of the three `mysql-X-X.yml` files to specify which version of MySQL server to start.
```

Comment:
Maybe something along these lines:

```suggestion
Ecco includes multiple Docker Compose definitions that can be used for this, one per supported MySQL version:

* docker-compose -f docker-mysql-base.yml -f mysql-5-6.yml up
* docker-compose -f docker-mysql-base.yml -f mysql-5-7.yml up
* docker-compose -f docker-mysql-base.yml -f mysql-8-0.yml up
```

(combined with my naming change proposals in Slack)

Reply:
Fixed: e7c3f96bf539f1bd30b401ffa7ce6385ec001376 :)